README.md

```diff
@@ -33,3 +34,9 @@ Once you run this command, you'll find the demo app at `localhost:8081` and Supe
 * **Type/Validation support** - Comprehensive type support using json-schema and custom validator function support for configuration values
 * **Multi-tenant support** - a single deployment allows multiple tenants to manage their configurations/experiments in a completely isolated manner
 * **Authn/Authz support** - control who can make configuration/experimentation changes
+
+## Interested to know more - reach out to the team !
+* Website: https://juspay.io/superposition
+* Twitter: https://x.com/superpositionjp
+* Discord: https://discord.gg/sBgYYUQU
+* Github Issues: https://github.com/juspay/superposition/issues
```

Comment:
@knutties we can use badges for these?
twitter:
https://img.shields.io/badge/X-000000?style=for-the-badge&logo=x&logoColor=white
discord:
https://img.shields.io/badge/Discord-5865F2?style=for-the-badge&logo=discord&logoColor=white
github:
https://img.shields.io/badge/GitHub-100000?style=for-the-badge&logo=github&logoColor=white

Reply:
@Datron - Just replace the labels with icons or even the links themselves ?  Or do we keep the icon and the label - atleast for X twitter seems still needed no ?